data/graphics.js

```diff
@@ -142,6 +142,30 @@ var BattleEffects = {
 		url: 'rightclaw.png',
 		w: 44, h: 60
 	},
+	leftslash: { // by Modeling Claw
+		url: 'leftslash.png',
+		w: 57, h: 56
+	},
+	rightslash: { // Modeling Claw
+		url: 'rightslash.png',
+		w: 57, h: 56
+	},
+	alpha: { // Ripped from Pokemon Global Link
+		url: 'alpha.png',
+		w: 80, h: 80
+	},
+	omega: { // Ripped from Pokemon Global Link
+		url: 'omega.png',
+		w: 80, h: 80
+	},
+	mega: { // http://orig05.deviantart.net/4ec3/f/2015/249/0/7/mega_evolution_templates_by_leparagon-d98ke9q.png
```

Comment:
Also you should ask for a specific license. "For anyone who wants to use them" is a good sign but legally we'd ideally need specific permission about what we can and can't do with it.